src/extensions/extension-loader.ts

```diff
@@ -151,7 +167,6 @@ export class ExtensionLoader {
   }
 
   loadOnClusterManagerRenderer() {
-    logger.info(`${logModule}: load on main renderer (cluster manager)`);
```

Comment:
Please don't remove these, I think that it is valuable logging information. (If you must, then change it to debug)

Reply:
Added them back as .debug